README.md

```diff
@@ -108,9 +108,10 @@ On iOS it is possible to configure the accessibility of the keychain by setting
  * AfterFirstUnlockThisDeviceOnly
  * WhenUnlocked
  * WhenUnlockedThisDeviceOnly
+ * WhenPasscodeSetThisDeviceOnly (this option is available only on iOS8 and later)
+but NOT to one of the following:
  * Always
  * AlwaysThisDeviceOnly
```

Comment:
Yes please change the PR and remove the insecure options all together, no point in having them there.